bin/cli.js

```diff
@@ -78,8 +78,9 @@ For more information, see https://webpack.js.org/api/cli/.`);
 				if (moduleName === "webpack") {
 					errorMessage = `\n${moduleName} not installed`;
 					instructions = `Install webpack to start bundling: \u001b[32m\n  $ npm install --save-dev ${moduleName}\n`;
-
-					if (process.env.npm_execpath !== undefined && process.env.npm_execpath.includes("yarn")) {
+					const path = require("path");
+					const fs = require("fs");
+					if (fs.existsSync(path.resolve(process.cwd(), "yarn.lock"))) {
```

Comment:
You can't use `existsSync` because it's not available on node `6.0.0` (check the history dropdown => https://nodejs.org/api/fs.html#fs_fs_existssync_path)